docs/source/tutorials/first_finetune_tutorial.rst

```diff
@@ -38,118 +44,120 @@ Then, it's as simple as:
 
 .. code-block:: bash
 
-  tune download \
-  meta-llama/Llama-2-7b \
-  --output-dir /tmp/llama2 \
-  --hf-token <ACCESS TOKEN>
+  tune download meta-llama/Llama-2-7b-hf \
+    --output-dir /tmp/Llama-2-7b-hf \
+    --hf-token <ACCESS TOKEN>
 
 This command will also download the model tokenizer and some other helpful files such as a Responsible Use guide.
 
-.. note::
-
-  You can also download the model directly through the Llama2 repository.
-  See `this page <https://llama.meta.com/get-started#getting-the-models>`_ for more details.
-
+|
 
 Selecting a recipe
 ------------------
 Recipes are the primary entry points for torchtune users.
-These can be thought of as end-to-end pipelines for training and optionally evaluating LLMs.
+These can be thought of as **singularly-focused scripts for interacting with LLMs**, including training
+inference, evaluation, and quantization.
 
 Each recipe consists of three components:
 
-* **Configurable parameters**, specified through yaml configs, command-line overrides and dataclasses
+* **Configurable parameters**, specified through yaml configs and command-line overrides
+* **Recipe script**, entry-point which puts everything together including parsing and validating configs, setting up the environment, and correctly using the recipe class
 * **Recipe class**, core logic needed for training, exposed to users through a set of APIs
```

Comment:
Sorry I'm a bit confused about the delineation between recipe script and recipe class here

Reply:
This is pulled from recipe deep-dive, so if we change here we need to change there.